app/api/resources/mentorship_relation.py

```diff
@@ -122,14 +122,14 @@ def put(cls, request_id):
 
         return response
 
-
 @mentorship_relation_ns.route('mentorship_relation/<int:request_id>/cancel')
 class CancelMentorshipRelation(Resource):
 
     @classmethod
     @jwt_required
     @mentorship_relation_ns.doc('cancel_mentorship_relation')
     @mentorship_relation_ns.expect(auth_header_parser)
```

Comment:
Remove this line because the next line does the same

Reply:
Oh right, forgot to remove it. Done!